composer.json

```diff
@@ -75,7 +71,7 @@
         "cobaltgrid/vatsim-stand-status": "~1.0",
         "doctrine/dbal": "~2.10",
         "guzzlehttp/guzzle": "~6.0",
-        "laravel/framework": "^7.0",
+        "laravel/framework": "7.23.2",
```

Comment:
This is restricting us to just this version? Wouldn't we want `^7.23` - not sure if there is a specific reason for locking to 7.23.2?

Reply:
There is a bug in 7.24 re the Spatie Permission package. Its being fixed on another PR but I've fixed the version so I can run this branch locally